src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml

```diff
@@ -1,122 +1,109 @@
-<dww:ModelessChildWindow x:Class="PythonNodeModelsWpf.ScriptEditorWindow"
-                         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-                         xmlns:fa="http://schemas.fontawesome.com/icons/"
-                         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-                         xmlns:ui1="clr-namespace:Dynamo.UI;assembly=DynamoCoreWpf"
-                         xmlns:dww="clr-namespace:Dynamo.Wpf.Windows;assembly=DynamoCoreWpf"
-                         xmlns:avalonedit="clr-namespace:ICSharpCode.AvalonEdit;assembly=ICSharpCode.AvalonEdit"
-                         xmlns:p="clr-namespace:PythonNodeModels.Properties;assembly=PythonNodeModels"
-                         xmlns:pw="clr-namespace:PythonNodeModelsWpf"
-                         xmlns:sys="clr-namespace:System;assembly=mscorlib"
-                         Title="{Binding NodeModel.Name, Mode=OneWay, RelativeSource={RelativeSource Self}}"
-                         Height="560"
-                         Width="600"
-                         MinWidth="500"
-                         MinHeight="450"
-                         ResizeMode="CanResizeWithGrip"
-                         Style="{DynamicResource DynamoWindowStyle}"
-                         AllowsTransparency="True"
-                         Background="Transparent"
-                         WindowStyle="None">
+<dww:ModelessChildWindow
+    x:Class="PythonNodeModelsWpf.ScriptEditorWindow"
```

Comment:
Why do we have changes to python script editor here? Looks like just alignment and white space changes. Are you using a different VS setting?